embedded-graphics/src/pixelcolor/mod.rs

```diff
@@ -88,11 +88,13 @@ mod conversion;
 mod gray_color;
 pub mod raw;
 mod rgb_color;
+pub mod web_colors;
```

Comment:
Does this need to be public? The module docs for `web_colors` could also be added to  `WebColors` instead IMO.